app/views/relationships/index.html.haml

```diff
@@ -0,0 +1,22 @@
+%section.people
+  %header
+    %h2 People I Follow
+  %table.table
+    %thead
+      %tr
+        %th(width="30%")
+        %th(width="20%") Videos in Queue
+        %th(width="20%") Followers
+        %th(width="30%") Unfollow
+    %tbody
+      - @following_relationships.each do |relationship|
```

Comment:
An optimization for looping is to use `find_each` instead of `each` (the syntax is exactly the same), which will instantiate the items in groups (of 1000 by default) rather than all items at once which can cause a server to run out of memory.